code/modules/mining/lavaland/ash_flora.dm

```diff
@@ -224,6 +224,39 @@
 	else
 		. = ..()
 
+//COYOTE - Wooden Bowls
+/obj/item/reagent_containers/glass/bowl/wooden_bowl
```

Comment:
Should have just made this in the coyote_modular folder but its fine being here <3